readme.md

```diff
@@ -71,15 +72,15 @@ Once installation is working, try creating and requesting resources. Note that t
 
 * **Your Answer:**
 
----
+The above code is taking the id: variable and making it into an objectID so the mongoDB knows the heirarchy of the DB.  Primary Key.
```

Comment:
This is not quite right. If you include the above in your code, it will require you to send an ObjectID when creating a new record.